runtime/hydradx/src/governance.rs

```diff
@@ -381,7 +381,7 @@ parameter_types! {
 	pub const TipReportDepositBase: Balance = 10 * DOLLARS;
 	pub const TipReportDepositPerByte: Balance = CENTS;
 	pub const MaximumReasonLength: u32 = 1024;
-	pub const MaxTipAmount: u128 = 1_000 * DOLLARS; // TODO: check this
+	pub const MaxTipAmount: u128 = 5_000_000 * UNITS; // ~$100k
```

Comment:
5_000_000 is more like ~$50k instead of ~$100k at current prices.

Reply:
we decided to keep the original value